native/src/components/Caption.tsx

```diff
@@ -1,7 +1,9 @@
 import React, { ReactElement } from 'react'
 import styled from 'styled-components/native'
 
-const H1 = styled.Text`
+import Text from './base/Text'
```

Comment:
Why `base` subfolder?

Reply:
i prefer to split up components a little bit and here we are overwriting ReactNative base components. I think we lose overview with this huge amount of components.